setup.py

```diff
@@ -46,14 +46,14 @@ def run(self):
 
 setup(
     install_requires=_REQUIREMENTS,
-    ext_modules=[
+    ext_modules=cythonize([
```

Comment:
Using `cythonize` rather than cython's `build_ext` seems to be more standard nowadays, and lets us set the language level directive.